.github/workflows/workflow.yml

```diff
@@ -1,8 +1,10 @@
 name: CI
 
 on:
-  - push
-  - pull_request
+  pull_request:
```

Comment:
I think you made the condition for pull requests empty.

Reply:
(remove the colon)

Reply:
Didn't make a difference. Anyway, I copied it from ocamllsp and it seems to work there.